example/16_gemm_reduce/gemm_reduce_xdl_max_fp16.cpp

```diff
@@ -37,20 +36,21 @@ using CLayout = ck::tensor_layout::gemm::RowMajor;
 using AElementOp  = ck::tensor_operation::element_wise::PassThrough;
```

Comment:
I'm looking an implementation where ```DDataType``` should be ```fp64```, and buffer atomic max for fp64 is used

Reply:
a771da2c6c3ed45ddb4e40a14a180e2c53c095ec